core/src/com/unciv/ui/overviewscreen/ReligionOverviewTable.kt

```diff
@@ -0,0 +1,117 @@
+        add(topButtonLabel).pad(5f)
+        addSeparator()
+        add(statsTable).pad(5f).row()
+        add(beliefsTable).pad(5f)
+    }
+    
+    private fun addReligionButtons() {
+        topButtons.clear()
+        val existingReligions: List<Religion> = gameInfo.civilizations.mapNotNull { it.religionManager.religion }
+        for (religion in existingReligions) {
+            val button: Button
+            if (religion.isPantheon()) {
+                val image = if (viewingPlayer.knows(religion.foundingCivName) || viewingPlayer.civName == religion.foundingCivName)
+                    ImageGetter.getNationIndicator(gameInfo.getCivilization(religion.foundingCivName).nation, 60f)
+                else
+                    ImageGetter.getRandomNationIndicator(60f)
+                button = Button(image, CameraStageBaseScreen.skin)
+            } else {
+                val image = ImageGetter.getReligionIcon(religion.iconName)
+                image.color = Color.BLACK
+                val icon = image.surroundWithCircle(60f)
+                button = Button(icon, CameraStageBaseScreen.skin)
+            }
+            
+            button.onClick {
+                selectedReligion = religion.name
+                draw(religion)
+            }
+            
+            if (selectedReligion == religion.name)
+                button.disable()
+            
+            topButtons.add(button).pad(5f)
+        }
+    }
+    
+    private fun loadReligion(religion: Religion) {
+        topButtonLabel.setText(religion.name.tr())
+        for (belief in religion.getPantheonBeliefs()) {
+            beliefsTable.add(createBeliefDescription(belief)).pad(10f).row()
+        }
+        for (belief in religion.getFollowerBeliefs()) {
+            beliefsTable.add(createBeliefDescription(belief)).pad(10f).row()
+        }
+        for (belief in religion.getFounderBeliefs()) {
+            beliefsTable.add(createBeliefDescription(belief)).pad(10f).row()
+        }
+        
+        statsTable.add("Religion Name:".tr())
+        statsTable.add(religion.name.tr()).pad(5f).row()
+        statsTable.add("Founding Civ:".tr())
+        val foundingCivName =
+            if (viewingPlayer.knows(religion.foundingCivName) || viewingPlayer.civName == religion.foundingCivName) 
+                religion.foundingCivName
+            else "???"
+        statsTable.add(foundingCivName.tr()).pad(5f).row()
+        statsTable.add("Cities following this religion:".tr())
+        statsTable.add(gameInfo.getCivilization(religion.foundingCivName).religionManager.citiesFollowingThisReligionCount().toString()).pad(5f).row()
+        
+        val minWidth = min(statsTable.minWidth, beliefsTable.minWidth) + 5f
+        
+        statsTable.width = minWidth
+        for (cell in beliefsTable.cells) {
+            cell.minWidth(minWidth)           
+        }
+    }
+    
+    private fun createBeliefDescription(belief: Belief): Table {
```

Comment:
Other ruleset objects, no longer all, have their getDescription(). If one _really_ wants to have "describe me" logic in one place only, a `getCivilopediaTextLines(ruleset).joinToString { it.text.tr() }` would mostly work. Not the first time I thought about a bitmask 'detailLevel' parameter...

*here* however I would _include_ the civilopedia output as it is, using the original renderer, with or without linking is a matter of taste.

Reply:
Try 
```kotlin
    private fun createBeliefDescription(belief: Belief) =
        MarkupRenderer.render(
            belief.run { sequence {
                yield(FormattedLine(name, size = 24, link = makeLink()))
                yield(FormattedLine())
                yieldAll(getCivilopediaTextLines(gameInfo.ruleSet))
            } }.toList(),
            noLinkImages = true
        ) {
            UncivGame.Current.setScreen(CivilopediaScreen(gameInfo.ruleSet, link = it))
        }.apply {
            background = ImageGetter.getBackground(ImageGetter.getBlue())
        }
```
with noLinkImages as is or without. The whole thing could be simpler without linking, and maybe I should wrap what the CivilopediaScreen does anyway _somewhere_ - but that's for later. Looks similar enough on screen. Just food for thought.

Reply:
<details><summary>omitting noLinkImages that looks like:</summary>

![image](https://user-images.githubusercontent.com/63000004/128645108-9cb55278-cde6-4788-ba4e-5c61965025cc.png)
</details>